From 1701d1f39863f5add115b9da3e5b27c68f8a9a0e Mon Sep 17 00:00:00 2001
From: Vineesh k k <vineesh.k.k@intel.com>
Date: Mon, 11 May 2015 10:21:26 +0530
Subject: [PATCH] EM/BQ24192_charger: Add wireless charging to BQ supported
 cables

Add new cable type=POWER_SUPPLY_CHARGER_TYPE_WIRELESS to
supported cables.

Change-Id: I93a6e218e67c3e8f0345ef2660ac9b8312199fb6
Tracked-On: https://jira01.devtools.intel.com/browse/AKP-1587
Signed-off-by: Vineesh k k <vineesh.k.k@intel.com>
---
 drivers/power/bq24192_charger.c | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/drivers/power/bq24192_charger.c b/drivers/power/bq24192_charger.c
index e4c8eaa61b16..10844bc1757d 100644
--- a/drivers/power/bq24192_charger.c
+++ b/drivers/power/bq24192_charger.c
@@ -370,6 +370,8 @@ static enum power_supply_type get_power_supply_type(
 		return POWER_SUPPLY_TYPE_USB_ACA;
 	case POWER_SUPPLY_CHARGER_TYPE_AC:
 		return POWER_SUPPLY_TYPE_MAINS;
+	case POWER_SUPPLY_CHARGER_TYPE_WIRELESS:
+		return POWER_SUPPLY_TYPE_WIRELESS;
 	case POWER_SUPPLY_CHARGER_TYPE_NONE:
 	case POWER_SUPPLY_CHARGER_TYPE_USB_SDP:
 	default:
@@ -2528,7 +2530,8 @@ struct bq24192_platform_data tbg24296_drvdata = {
 	.supplied_to = bq24192_supplied_to,
 	.num_throttle_states = ARRAY_SIZE(byt_throttle_states),
 	.num_supplicants = ARRAY_SIZE(bq24192_supplied_to),
-	.supported_cables = POWER_SUPPLY_CHARGER_TYPE_USB,
+	.supported_cables = (POWER_SUPPLY_CHARGER_TYPE_USB |
+			POWER_SUPPLY_CHARGER_TYPE_WIRELESS),
 	.sfi_tabl_present = true,
 	.max_cc = 1800,	/* 1800 mA */
 	.max_cv = 4350,	/* 4350 mV */
-- 
1.9.1

